app/models/device.rb

```diff
@@ -1,7 +1,4 @@
 class Device < ApplicationRecord
-  # @fdoxyz to remove app_bundle from Device soon
```

Comment:
@fdoxyz This comment was from April. I checked with Blazer that we indeed have no data stored in this column and would remove it in this PR if that's ok for you.

Reply:
Sounds great, thanks for looping back here for all these cleanups! 🙌